oracle/blockoracle.go

```diff
@@ -64,7 +64,9 @@ func (bo *MinerBlockOracle) BlockEligible(layerID types.LayerID) (types.ATXID, [
 		return types.ATXID{}, nil, fmt.Errorf("cannot calc eligibility, not synced yet")
 	}
 	epochNumber := layerID.GetEpoch(bo.layersPerEpoch)
-	bo.log.Info("asked for eligibility for epoch %d (cached: %d)", epochNumber, bo.proofsEpoch)
+	bo.log.With().Info("asked for eligibility for epoch",
+		log.EpochID(uint64(epochNumber)),
```

Comment:
in continuation to @y0sher's comment, since there are two epoch ids in this log message, there's no way to simplify the other one, but this one should actually be just:
```suggestion
		epochNumber,
```

The `LoggableField` interface means you can just include the field here as-is.

Reply:
ops, sorry for missing that, still getting used to this.

Reply:
I've created this PR: https://github.com/spacemeshos/go-spacemesh/pull/2051

It makes the code more consistent and less confusing by getting rid of all the legacy logging methods.